plugins/ScheduledReports/vue/src/ManageScheduledReport/ManageScheduledReport.vue

```diff
@@ -218,12 +220,13 @@ export default defineComponent({
       showReportsList: true,
       report: {} as unknown as Report,
       selectedReports: {},
+      sendingReports: [],
     };
   },
   methods: {
     sendReportNow(idReport: string|number) {
       scrollToTop();
-
+      this.sendingReports.push(idReport);
```

Comment:
Is `idReport` always numeric? If so, can we type it to an int here and then in the finally block of the ajax call we don't have to accept both `string | number` types, where we do a string comparison.

Reply:
@michalkleiner I tried, it was giving too many vue errors even after converting it to an int, I would like to pass this one as its not creating any harm.